include/boost/math/tools/random_vector.hpp

```diff
@@ -35,6 +35,26 @@ std::vector<T> generate_random_vector(std::size_t size, std::size_t seed)
     return v;
 }
 
+template<typename T, typename std::enable_if<std::is_floating_point<T>::value, bool>::type = true>
+std::vector<T> generate_random_vector(std::size_t size, std::size_t seed, T min_val, T max_val)
```

Comment:
Note that the `min_val, max_val` is misleading here: It's passed into a normal distribution where min_val = mean and max_val = std_dev.